tests/utils/test_add_new_model_like.py

```diff
@@ -836,7 +843,7 @@ def test_retrieve_info_for_model_with_wav2vec2(self):
         ]
         expected_model_classes = {
             "pt": set(wav2vec2_classes),
-            "tf": {f"TF{m}" for m in wav2vec2_classes[:1]},
+            "tf": {f"TF{m}" for m in wav2vec2_classes[:1] + ["Wav2Vec2ForSequenceClassification"]},
```

Comment:
I think it'd be better to still use the `wav2vec2_classes` object so something like: 

```suggestion
            "tf": {f"TF{m}" for m in [wav2vec2_classes[0], wav2vec2_classes[-2]]},
```

Reply:
Done 👍🏻

Reply:
Thanks!